src/content/edit-user-script.js

```diff
@@ -1,5 +1,7 @@
 // TODO: Search, replace.
-// TODO: Put name in title.
+
+// Change this title as soon as possible, but it won't change later.
+document.getElementById('save').setAttribute('title', _('Save'));
```

Comment:
It's probably worth using rivets, like in the rest of our HTML?  I'm worried about one-offs like this piling up over time.

Reply:
I don't know too much about rivets, but does that translate anything other than `textContent`? You know, when I have `<tag>{Save|i18n}</tag>`, that's not a problem, but with `<tag text="{Save|i18n}">...</tag>`, it should fail. And "__MSG_Save__" generally doesn't work in `html` files as well.

Reply:
Another remark: This will probably move to the menu bar later, and when in `src/content/menu-bar/menu-bar.run.js`, it will be handled much easier:
```
addItem('execCommand', { text: _('Save'), cmd: 'save', key: 's' })
```

Reply:
Oh, yeah I'm not sure that/how it will work in an attribute.  Okay!

Reply:
Pretty sure `<tag text=" 'Save' | i18n ">...</tag>` will work.

Reply:
Er, might need `rv-text` as the attribute name.

Reply:
> Pretty sure <tag text=" 'Save' | i18n ">...</tag> will work.

Tried this:
```
< div id="save" class="command-item" title=" 'Save' | i18n " >
```
That didn't work: it showed  just `'Save' | i18n`!

Reply:
With `rv-` ? I remember reading this somewhere and was like "Oh that's cool". But I never noted down the exact syntax.

Reply:
Trying sth. with `rv-title=`...

Reply:
@Sxderp Oops, overlooked your comment! So you've had the same idea. If I read the documents correctly, we should use a prefix `rv-` in front of the attribute `title` and specify the value `value | formatter`, which is obviously `'Save'|i18n` in this case. But it reacts, as if there was no `title="Save"` at all, it's just ignored...

Reply:
Make sure you
- Add `rivets.bundled.min.js` (or whatever it's called) to the `.html`
- Add `rivetsformatters.js` (or whatever it's called) to the `.html`
- Add `i18n.js` to the `.html`
- Put `rivets.bind(document, {});` somewhere in the `.js`

Assuming I didn't miss anything it should work then. I did a very minimal test with some one-off HTML page.

Reply:
Great! I forgot the binding. But it works now!